### PR TITLE
Feature/SfChevron

### PR DESCRIPTION
**Pull request by Michal-Dziedzinski**
# Scope of work

Add chevron component, which can be use for example in Accordion component

# Screenshots of visual changes
chevron

# Checklist

- [x] I followed [composition rules](https://github.com/DivanteLtd/storefront-ui/blob/master/docs/component-rules.md) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/shared/styles/components/SfChevron.scss [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+$chevron-duration: .5s !default;
+$chevron-easing: cubic-bezier(.25, 1.7, .35, .8) !default;
+$chevron-size-desktop: var(--chevron-size) !default;
+$chevron__bar-width: calc(#{$chevron-size-desktop} / 2) !default;
+$chevron__bar-height: calc(#{$chevron__bar-width} / 4) !default;
+
+$chevron__bar-top-desktop: calc(calc(#{$chevron-size-desktop} / 2) - calc(#{$chevron__bar-height} / 2)) !default;
+$chevron__bar-left-right-desktop: calc(#{$chevron-size-desktop} / 9) !default;
+$chevron__bar-rotate: 45deg !default;
+$chevron__bar-color: var(--chevron-color);
```
Comment (leomp12) [inline packages/shared/styles/components/SfChevron.scss]: Imo you should preset with some color, then you can use this color or replace with the CSS var (when browser has support and respective var is defined), wdyt?

```suggestion
$chevron__bar-color: $c-primary !default;
```

File: packages/shared/styles/components/SfChevron.scss [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    width: $chevron__bar-width;
+    height: $chevron__bar-height;
+    display: block;
+    transform: rotate($chevron__bar-rotate);
+    float: right;
+    border-radius: 50%;
+
+    &:after {
+      content: "";
+      background-color: $chevron__bar-color;
```
Comment (leomp12) [inline packages/shared/styles/components/SfChevron.scss]:
```suggestion
      background-color: $chevron__bar-color;
      background-color: var(--chevron-color);
```

File: packages/vue/src/components/atoms/SfChevron/SfChevron.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      default: false
+    },
+    /** Color of Chevron. Default is black.  */
+    color: {
+      type: String,
+      default: "#000000"
+    },
+    /** Size of Chevron component. Default is 40px.  */
+    size: {
+      type: String,
```
Comment (leomp12) [inline packages/vue/src/components/atoms/SfChevron/SfChevron.js]: In general, color and size customization should be done with CSS modifiers or SCSS vars, not by props.
I think it's a convention for all components...

File: packages/vue/src/components/atoms/SfChevron/SfChevron.html [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<div class="sf-chevron" :style="getStyle">
+  <slot>
```
Comment (przspa) [inline packages/vue/src/components/atoms/SfChevron/SfChevron.html]:
```suggestion
  <slot v-bind="{isActive}">
```
@Michal-Dziedzinski nice to need when isActive inside slot :)

**Comment by filrak** [comment]
part of https://github.com/DivanteLtd/storefront-ui/pull/264

**Pull request closed without merging**